how-to/containerd-kata.md

```diff
@@ -0,0 +1,345 @@
+
+Before the Runtime was introduced, Kubernetes isn't aware of the difference of runtimes on the node. `kubelet`
+creates Pod sandboxes and containers through CRI implementations and treat all the Pods equally. However, there
+are requirements to run trusted Pod (such as kubernetes plugin) in native container like runC and run untrusted
+workload with isolated sandboxes such as Kata Containers.
+
+As a result, the CRI implementations extended their semanitics for the requirements:
+
+- At the beginning, [frakti](https://github.com/kubernetes/frakti) check the network configuration of a Pod, and
+  treat Pod with `host` network as trusted, while others as untrusted;
+- The containerd introduced an annotation for untrusted Pod since [v1.0](https://github.com/containerd/cri/blob/v1.0.0-rc.0/docs/config.md):
+  ```yaml
+  annotations:
+     io.kubernetes.cri.untrusted-workload: "true"
+  ```
+- Similarly, CRI-O introduced the annotation `io.kubernetes.cri-o.TrustedSandbox` for the same case.
+
+To eliminate the complexity of user configuration introduced by the non-standardized annotations and provide 
+extensibility, `RuntimeClass` was introduced and users could affect the runtime behavior through it without 
+the knowledge of the CRI daemons. We suggest the users have multiple runtimes use `RuntimeClass` instead of 
+the deprecated annotations.
+
+### Containerd Runtime V2 API (Shim V2 API)
+
+The [`containerd-shim-kata-v2` (short as `shimv2` in this documentation)](https://github.com/kata-containers/runtime/tree/master/containerd-shim-v2) 
+implements the [Containerd Runtime V2 (Shim API)](https://github.com/containerd/containerd/tree/master/runtime/v2) for Kata.
+With `shimv2`, kubernetes could launch Pod and OCI compatible containers with one shim (the `shimv2`) per Pod instead
+of `2N+1` shims (a `containerd-shim` and a `kata-shim` for each container and the Pod sandbox itself), and no standalone
+`kata-proxy` process even if no vsock available.
+
+![kubernetes integration with shimv2](../arch-images/shimv2.svg)
+
+The shim v2 is introduced in containerd [v1.2.0](https://github.com/containerd/containerd/releases/tag/v1.2.0) and kata shimv2
+is implemented in Kata Containers v1.5.0.
+
+## Install
+
+### Install Kata Containers
+
+Follow the instructions to [install Kata Containers](https://github.com/kata-containers/documentation/blob/master/install/README.md).
+
+### Install containerd with cri plugin
+
+> **Note:** `cri` is a native plugin of containerd 1.1 and above. It is built into containerd and enabled by default. 
+> You don't need to install `cri` if you have containerd 1.1 or above. Just check if the `cri` plugin has been disabled 
+> in the containerd configuration file.
+
+Follow the instructions from the [CRI installation guide](http://github.com/containerd/cri/blob/master/docs/installation.md).
+
+And check if `containerd` is now available
+  
+```bash
+> command -v containerd
```

Comment:
Could you change the prompt to `$` for consistency with https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#code-blocks.

```sh

    ```bash
    $ command -v containerd
    ```
```

Same comment for commands below...